routes/docs/index.js

```diff
@@ -1,15 +1,33 @@
 const i18n = require('../../lib/i18n')
+const cheerio = require('cheerio')
 
 module.exports = (req, res) => {
   const docsReadme = i18n.docs[req.language]['/docs/README']
 
+  // Temporary fix to collect all section HTML
+  // Replace with direct HTML when electron-i18n format changes
+  const html = docsReadme.sections.reduce(
```

Comment:
Is there a way we can use the final HTML if it's already available or use this fix if it isn't? Maybe by checking if `sections` exists? 
That way we avoid any possible downtime between upgrading the `i18n` package and removing this (or having to do changes to the PR that updates `i18n`).